index.html

```diff
@@ -5,6 +5,9 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Gemunu+Libre:wght@500&family=Indie+Flower&display=swap" rel="stylesheet">
```

Comment:
One alternative way to import fonts is to [download the fonts](https://fonts.google.com/), place them into a "fonts" folder, and use `@font-face` in CSS. For example:
```css
@font-face {
  font-family: 'Gemunu Libre';
  font-weight: 500;
  src: local('Gemunu Libre'), url('./fonts/Gemunu_Libre/Gemunu_Libre-VariableFont_wght.ttf') format('ttf');
}

@font-face {
  font-family: 'Indie Flower';
  src: local('Indie Flower'), url('./fonts/Indie_Flower/Indie_Flower-Regular.ttf') format('ttf');
}
```